django_mongodb/base.py

```diff
@@ -73,6 +91,13 @@ class DatabaseWrapper(BaseDatabaseWrapper):
         "regex": lambda val: re.compile(val),
         "iregex": lambda val: re.compile(val, re.IGNORECASE),
     }
+    mongo_aggregates = {
```

Comment:
🤔 In this PR add support for annotates, so Why is mongo_aggregates the name? shouldn't it be mongo_annotates? IMO aggregates sounds like $sum, $avg, and all this kind of group by operators.

Reply:
Originally I called it `mongo_functions` but renamed it because https://www.mongodb.com/docs/manual/reference/operator/aggregation/lt/ describes it as "aggregation". Maybe it should be `mongo_aggregations`. I'm not sure the precise difference between aggregate and aggregation!